src/index.html

```diff
@@ -594,27 +594,27 @@ <h4 class="h1 heading-title">Deep learning insights as never seen before!</h4>
                     <div class="row">
                         <div class="col-lg-6 col-md-12 col-sm-12 col-xs-12">
                             <div class="heading">
-                                <p class="heading-subtitle">Maturify is in auto-pilot, there is nothing much for admins to get involved.
+                                <p class="heading-subtitle">Maturify serves as an auto-pilot, making it unnecessary for admins to get involved.
```

Comment:
saying 'unnecessary' might be misleading . I feel we can say something like 'bringing the admins involvement to a minimum'. What do you think?

Reply:
Good idea! Done!